packages/vscode-extension/src/handlers.ts

```diff
@@ -280,7 +313,7 @@ export async function migrateV1ProjectHandler(args?: any[]): Promise<Result<null
   );
   const result = await runCommand(Stage.migrateV1);
   await openMarkdownHandler();
-  await vscode.commands.executeCommand("setContext", "fx-extension.sidebarWelcome", false);
+  await vscode.commands.executeCommand("setContext", "fx-extension.sidebarWelcome.treeview", false);
```

Comment:
Do we need to set `fx-extension.sidebarWelcome.default` / `fx-extension.sidebarWelcome.top` / `fx-extension.sidebarWelcome.buttom` to false?